tests/flake8_typing_imports_test.py

```diff
@@ -118,3 +118,76 @@ def test_missing_overload_define():
         )
     with version_ctx(Version(3, 5, 2)):
         assert not results('from typing import overload')
+
+
+@pytest.mark.parametrize(
+    's', [
```

Comment:
prefer tuples to lists when equivalent: https://github.com/asottile/scratch/wiki/Prefer-tuples-to-lists